code_samples/api/public_php_api/src/Command/FindContentTypeCommand.php

```diff
@@ -0,0 +1,55 @@
+<?php declare(strict_types=1);
+
+namespace App\Command;
+
+use Ibexa\Contracts\Core\Repository\ContentTypeService;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\Query\ContentTypeQuery;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\Query\Criterion;
+use Ibexa\Contracts\Core\Repository\Values\ContentType\Query\SortClause;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+#[AsCommand(
+    name: 'doc:find_content_types',
+    description: 'Lists content types that match specific criteria.'
+)]
```

Comment:
Friendly reminder that this will not work on 4.6 - and needs to be downgraded (I can help with that if needed).

In general it's easier to write code for 4.6 and then update it to v5 with Rector - we don't have Rector downgrades configured (if they are doable at all)